llm_on_ray/finetune/ds_llama2_ds_zero3_config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "steps_per_print": 64,
+    "train_batch_size": "auto",
+    "train_micro_batch_size_per_gpu": "auto",
+    "gradient_accumulation_steps": "auto",
+    "bf16": {
+        "enabled": true
+    },
+    "optimizer": {
```

Comment:
We already have optimizer configured in the yaml, why it appears again in this json file?

Reply:
"optimizer" must be specified in json config file.